ow_lander/scripts/mechanical_power_arm.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python2
-
```

Comment:
I support leaving an empty line between the environment and the imports

Reply:
OK!